pkg/internal/tracking/query.go

```diff
@@ -0,0 +1,28 @@
+package tracking
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+func AppendMetadataToSql(sql string, metadata Metadata) (string, error) {
```

Comment:
nit: simply AppendMetadata and Parse/ExtractMetadata would be enough

Reply:
Renamed